Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM mcr.microsoft.com/dotnet/core/aspnet:3.1.3-bionic
```

Comment:
Waarom bouwen we hem in de published container? Dit doen we bij andere services niet.